2017-10-07 15:25:17,653 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-07 15:25:17,653 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-07 15:44:28,137 - ERROR - Got signal 2 in MainProcess/MainThread
2017-10-07 15:44:28,137 - ERROR - Got signal 2
2017-10-07 15:45:06,630 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-07 15:45:06,630 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-07 15:45:09,536 - ERROR - list index out of range
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\api.py", line 1058, in _dispatch
    return self._handle_request(method, params)
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\api.py", line 1045, in _handle_request
    return self.handlers[method](self, params)
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\api.py", line 442, in HandleJoinChan
    if addressGeneratorReturnValue[0] == 'chan name does not match address':
IndexError: list index out of range
2017-10-07 15:46:05,493 - ERROR - 85.114.135.102:8444 error: 2, Server full, please try again later.
2017-10-07 15:46:30,852 - ERROR - 185.158.248.216:8444 error: 2, Too many connections from your IP. Closing connection.
2017-10-07 15:50:27,979 - ERROR - Got signal 2 in MainProcess/MainThread
2017-10-07 15:50:27,979 - ERROR - Got signal 2
2017-10-07 15:50:34,079 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-07 15:50:34,079 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-07 15:50:37,392 - ERROR - list index out of range
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\api.py", line 1058, in _dispatch
    return self._handle_request(method, params)
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\api.py", line 1045, in _handle_request
    return self.handlers[method](self, params)
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\api.py", line 442, in HandleJoinChan
    if addressGeneratorReturnValue[0] == 'chan name does not match address':
IndexError: list index out of range
2017-10-07 15:50:59,951 - ERROR - Got signal 2 in MainProcess/MainThread
2017-10-07 15:50:59,951 - ERROR - Got signal 2
2017-10-07 15:51:00,099 - ERROR - Fast PoW got StopIteration
2017-10-07 15:54:23,075 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-07 15:54:23,075 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-07 16:01:03,507 - ERROR - Got signal 2 in MainProcess/MainThread
2017-10-07 16:01:03,507 - ERROR - Got signal 2
2017-10-07 16:29:10,009 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-07 16:29:10,009 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-07 16:29:24,362 - ERROR - Got signal 2 in MainProcess/MainThread
2017-10-07 16:29:24,362 - ERROR - Got signal 2
2017-10-07 16:29:24,410 - ERROR - Fast PoW got StopIteration
2017-10-07 16:29:33,956 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-07 16:29:33,956 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-07 16:30:21,680 - ERROR - Got signal 2 in MainProcess/MainThread
2017-10-07 16:30:21,680 - ERROR - Got signal 2
2017-10-07 16:30:21,930 - ERROR - Fast PoW got StopIteration
2017-10-07 16:34:04,730 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-07 16:34:04,730 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-07 20:20:44,187 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-07 20:20:44,188 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-07 20:23:05,209 - ERROR - Fast PoW got StopIteration
2017-10-07 20:24:12,661 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-07 20:24:12,663 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-07 20:44:58,388 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-07 20:44:58,390 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-07 20:45:49,493 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-07 20:45:49,494 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-07 20:52:55,301 - ERROR - Got signal 2 in MainProcess/MainThread
2017-10-07 20:52:55,301 - ERROR - Got signal 2
2017-10-07 20:53:24,030 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-07 20:53:24,032 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-07 20:55:26,585 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-07 20:55:26,585 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-07 20:59:37,003 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-07 20:59:37,003 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-07 21:00:37,119 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-07 21:00:37,119 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-07 21:02:02,630 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-07 21:02:02,630 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-07 21:02:25,631 - ERROR - Fast PoW got StopIteration
2017-10-07 21:07:54,232 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-07 21:07:54,246 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-07 21:08:17,436 - ERROR - Fast PoW got StopIteration
2017-10-07 21:08:57,160 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-07 21:08:57,160 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-07 21:10:10,976 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-07 21:10:10,976 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-07 21:10:34,411 - ERROR - Fast PoW got StopIteration
2017-10-08 22:07:43,062 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-08 22:07:43,062 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-08 22:08:25,035 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-08 22:08:25,035 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-08 22:35:09,117 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-08 22:35:09,117 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-08 22:42:03,200 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-08 22:42:03,200 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-08 22:43:09,174 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-08 22:43:09,176 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-08 22:45:19,983 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-08 22:45:19,983 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-08 22:48:00,308 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-08 22:48:00,309 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-08 22:50:09,368 - ERROR - Got signal 2 in MainProcess/MainThread
2017-10-08 22:50:09,368 - ERROR - Got signal 2
2017-10-08 22:50:45,596 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-08 22:50:45,596 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-08 22:51:26,043 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-08 22:51:26,045 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-08 22:53:35,599 - ERROR - Got signal 2 in MainProcess/MainThread
2017-10-08 22:53:35,599 - ERROR - Got signal 2
2017-10-08 22:53:41,915 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-08 22:53:41,917 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-08 22:54:00,400 - ERROR - Got signal 2 in MainProcess/MainThread
2017-10-08 22:54:00,400 - ERROR - Got signal 2
2017-10-08 22:54:03,232 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-08 22:54:03,232 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-08 22:55:22,119 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-08 22:55:22,121 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-08 23:05:45,707 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-08 23:05:45,710 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-08 23:08:22,326 - ERROR - Got signal 2 in MainProcess/MainThread
2017-10-08 23:08:22,326 - ERROR - Got signal 2
2017-10-08 23:09:25,730 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-08 23:09:25,732 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-08 23:14:41,890 - ERROR - Got signal 2 in MainProcess/MainThread
2017-10-08 23:14:41,891 - ERROR - Got signal 2
2017-10-08 23:15:03,780 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-08 23:15:03,782 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-08 23:15:56,917 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-08 23:15:56,917 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-08 23:18:15,719 - ERROR - Got signal 2 in MainProcess/MainThread
2017-10-08 23:18:15,720 - ERROR - Got signal 2
2017-10-08 23:18:37,821 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-08 23:18:37,822 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-08 23:28:05,983 - ERROR - Got signal 2 in MainProcess/MainThread
2017-10-08 23:28:05,983 - ERROR - Got signal 2
2017-10-09 21:23:42,923 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-09 21:23:42,924 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-10 00:05:19,875 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-10 00:05:19,890 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-10 00:09:26,640 - ERROR - Got signal 2 in MainProcess/MainThread
2017-10-10 00:09:26,640 - ERROR - Got signal 2
2017-10-10 00:09:31,581 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-10 00:09:31,582 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-10 01:04:16,901 - ERROR - Got signal 2 in MainProcess/MainThread
2017-10-10 01:04:16,901 - ERROR - Got signal 2
2017-10-10 01:04:20,648 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-10 01:04:20,648 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-10 01:21:19,885 - ERROR - Got signal 2 in MainProcess/MainThread
2017-10-10 01:21:19,885 - ERROR - Got signal 2
2017-10-10 01:21:29,736 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-10 01:21:29,736 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-10 20:02:38,114 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-10 20:02:38,115 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-10 20:08:02,793 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-10 20:08:02,795 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-10 21:23:25,589 - ERROR - Got signal 2 in MainProcess/MainThread
2017-10-10 21:23:25,591 - ERROR - Got signal 2
2017-10-10 21:24:25,983 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-10 21:24:25,984 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-10 21:26:00,694 - ERROR - Got signal 2 in MainProcess/MainThread
2017-10-10 21:26:00,694 - ERROR - Got signal 2
2017-10-10 21:26:17,467 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-10 21:26:17,469 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-10 21:27:28,773 - ERROR - Got signal 2 in MainProcess/MainThread
2017-10-10 21:27:28,775 - ERROR - Got signal 2
2017-10-10 21:45:11,933 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-10 21:45:11,934 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-10 21:56:09,181 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-10 21:56:09,183 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-10 21:56:46,967 - ERROR - Got signal 2 in MainProcess/MainThread
2017-10-10 21:56:46,967 - ERROR - Got signal 2
2017-10-10 21:56:53,482 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-10 21:56:53,483 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-10 21:57:18,805 - ERROR - Got signal 2 in MainProcess/MainThread
2017-10-10 21:57:18,805 - ERROR - Got signal 2
2017-10-10 21:57:30,424 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-10 21:57:30,424 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-10 22:08:49,855 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-10 22:08:49,857 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-10 22:59:26,030 - ERROR - Got signal 2 in MainProcess/MainThread
2017-10-10 22:59:26,032 - ERROR - Got signal 2
2017-10-10 23:49:26,321 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-10 23:49:26,321 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-11 00:28:37,585 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-11 00:28:37,586 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-11 01:31:29,773 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-11 01:31:29,775 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-11 01:37:11,463 - ERROR - Got signal 2 in MainProcess/MainThread
2017-10-11 01:37:11,463 - ERROR - Got signal 2
2017-10-11 01:37:17,641 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-11 01:37:17,641 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-11 02:48:51,361 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-11 02:48:51,361 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-11 03:07:33,489 - ERROR - Got signal 2 in MainProcess/MainThread
2017-10-11 03:07:33,489 - ERROR - Got signal 2
2017-10-11 03:10:42,687 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-11 03:10:42,687 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-11 03:11:32,392 - ERROR - Got signal 2 in MainProcess/MainThread
2017-10-11 03:11:32,392 - ERROR - Got signal 2
2017-10-11 03:11:35,170 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-11 03:11:35,171 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-11 03:32:25,983 - ERROR - Got signal 2 in MainProcess/MainThread
2017-10-11 03:32:25,983 - ERROR - Got signal 2
2017-10-11 03:35:28,976 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-11 03:35:28,976 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-11 03:52:16,950 - ERROR - Got signal 2 in MainProcess/MainThread
2017-10-11 03:52:16,950 - ERROR - Got signal 2
2017-10-11 03:52:17,668 - ERROR - Got signal 2 in MainProcess/MainThread
2017-10-11 03:52:17,668 - ERROR - Got signal 2
2017-10-11 03:52:37,538 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-11 03:52:37,539 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-11 13:26:07,187 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-11 13:26:07,188 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-11 14:15:41,845 - ERROR - Got signal 2 in MainProcess/MainThread
2017-10-11 14:15:41,845 - ERROR - Got signal 2
2017-10-11 14:15:51,891 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-11 14:15:51,892 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-11 23:17:32,006 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-11 23:17:32,007 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-12 00:31:46,084 - ERROR - Got signal 2 in MainProcess/MainThread
2017-10-12 00:31:46,084 - ERROR - Got signal 2
2017-10-12 00:31:47,099 - ERROR - Got signal 2 in MainProcess/MainThread
2017-10-12 00:31:47,099 - ERROR - Got signal 2
2017-10-12 00:33:09,456 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-12 00:33:09,456 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-12 21:44:01,598 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-12 21:44:01,598 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-12 23:59:44,068 - ERROR - Got signal 2 in MainProcess/MainThread
2017-10-12 23:59:44,069 - ERROR - Got signal 2
2017-10-13 00:00:24,186 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-13 00:00:24,187 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-13 00:02:17,605 - ERROR - Got signal 2 in MainProcess/MainThread
2017-10-13 00:02:17,605 - ERROR - Got signal 2
2017-10-13 00:02:21,019 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-13 00:02:21,019 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-13 00:59:52,680 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-13 00:59:52,681 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-13 01:34:31,480 - ERROR - Got signal 2 in MainProcess/MainThread
2017-10-13 01:34:31,480 - ERROR - Got signal 2
2017-10-13 01:36:06,668 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-13 01:36:06,670 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-13 02:01:34,062 - ERROR - Got signal 2 in MainProcess/MainThread
2017-10-13 02:01:34,062 - ERROR - Got signal 2
2017-10-13 02:01:50,832 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-13 02:01:50,834 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-13 15:26:52,559 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-13 15:26:52,561 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-13 16:01:57,986 - ERROR - Got signal 2 in MainProcess/MainThread
2017-10-13 16:01:57,986 - ERROR - Got signal 2
2017-10-13 18:10:12,165 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-13 18:10:12,167 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-13 18:10:41,463 - ERROR - Got signal 2 in MainProcess/MainThread
2017-10-13 18:10:41,463 - ERROR - Got signal 2
2017-10-13 18:14:22,065 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-13 18:14:22,065 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-13 22:33:50,298 - ERROR - Got signal 2 in MainProcess/MainThread
2017-10-13 22:33:50,298 - ERROR - Got signal 2
2017-10-13 22:33:54,128 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-13 22:33:54,128 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-14 22:25:30,809 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-14 22:25:30,811 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-14 23:12:14,473 - ERROR - Got signal 2 in MainProcess/MainThread
2017-10-14 23:12:14,473 - ERROR - Got signal 2
2017-10-14 23:12:19,634 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-14 23:12:19,634 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-14 23:48:53,687 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-14 23:48:53,688 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-15 00:02:24,901 - ERROR - Got signal 2 in MainProcess/MainThread
2017-10-15 00:02:24,901 - ERROR - Got signal 2
2017-10-15 00:02:31,315 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-15 00:02:31,316 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-15 17:02:46,934 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-15 17:02:46,936 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-15 17:48:31,299 - ERROR - Got signal 2 in MainProcess/MainThread
2017-10-15 17:48:31,301 - ERROR - Got signal 2
2017-10-15 17:48:34,086 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-15 17:48:34,088 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-15 19:03:17,055 - ERROR - Got signal 2 in MainProcess/MainThread
2017-10-15 19:03:17,055 - ERROR - Got signal 2
2017-10-15 19:03:28,220 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-15 19:03:28,221 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-15 20:13:43,657 - ERROR - Got signal 2 in MainProcess/MainThread
2017-10-15 20:13:43,658 - ERROR - Got signal 2
2017-10-15 20:13:44,293 - ERROR - Got signal 2 in MainProcess/MainThread
2017-10-15 20:13:44,293 - ERROR - Got signal 2
2017-10-15 20:15:01,992 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-15 20:15:01,993 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-15 23:54:58,592 - ERROR - Got signal 2 in MainProcess/MainThread
2017-10-15 23:54:58,592 - ERROR - Got signal 2
2017-10-15 23:55:17,299 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-15 23:55:17,299 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-15 23:55:22,176 - ERROR - Got signal 2 in MainProcess/MainThread
2017-10-15 23:55:22,176 - ERROR - Got signal 2
2017-10-15 23:55:36,786 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-15 23:55:36,788 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-16 00:41:41,956 - ERROR - Got signal 2 in MainProcess/MainThread
2017-10-16 00:41:41,957 - ERROR - Got signal 2
2017-10-16 00:41:50,362 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-16 00:41:50,365 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-16 00:45:33,056 - ERROR - Got signal 2 in MainProcess/MainThread
2017-10-16 00:45:33,056 - ERROR - Got signal 2
2017-10-16 00:47:19,374 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-16 00:47:19,375 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-16 00:59:10,890 - ERROR - Got signal 2 in MainProcess/MainThread
2017-10-16 00:59:10,890 - ERROR - Got signal 2
2017-10-16 01:04:46,507 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-16 01:04:46,509 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-16 01:06:53,752 - ERROR - Got signal 2 in MainProcess/MainThread
2017-10-16 01:06:53,753 - ERROR - Got signal 2
2017-10-16 01:06:54,752 - ERROR - Got signal 2 in MainProcess/MainThread
2017-10-16 01:06:54,752 - ERROR - Got signal 2
2017-10-16 01:06:55,164 - ERROR - Got signal 2 in MainProcess/MainThread
2017-10-16 01:06:55,165 - ERROR - Got signal 2
2017-10-16 01:07:18,213 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-16 01:07:18,214 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-16 01:26:38,115 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-16 01:26:38,115 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-16 01:48:28,734 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-16 01:48:28,742 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-16 01:48:29,211 - ERROR - There was a problem testing for a Namecoin daemon. Hiding the Fetch Namecoin ID button
2017-10-16 01:48:29,470 - WARNING - No notification.message plugin found
2017-10-16 18:11:12,558 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-16 18:11:12,559 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-16 18:11:12,994 - ERROR - There was a problem testing for a Namecoin daemon. Hiding the Fetch Namecoin ID button
2017-10-16 18:11:13,049 - WARNING - No notification.message plugin found
2017-10-16 18:12:02,713 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-16 18:12:02,713 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-16 18:12:03,065 - ERROR - There was a problem testing for a Namecoin daemon. Hiding the Fetch Namecoin ID button
2017-10-16 18:12:03,107 - WARNING - No notification.message plugin found
2017-10-16 18:12:36,874 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-16 18:12:36,875 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-16 18:20:23,525 - ERROR - Got signal 2 in MainProcess/MainThread
2017-10-16 18:20:23,525 - ERROR - Got signal 2
2017-10-16 18:20:23,648 - ERROR - Fast PoW got StopIteration
2017-10-16 19:33:08,025 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-16 19:33:08,026 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-17 01:23:34,071 - ERROR - Got signal 2 in MainProcess/MainThread
2017-10-17 01:23:34,072 - ERROR - Got signal 2
2017-10-17 01:24:25,661 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-17 01:24:25,663 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-17 01:25:46,983 - ERROR - Got signal 2 in MainProcess/MainThread
2017-10-17 01:25:46,983 - ERROR - Got signal 2
2017-10-17 01:25:49,584 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-17 01:25:49,585 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-17 01:35:00,994 - ERROR - Got signal 2 in MainProcess/MainThread
2017-10-17 01:35:00,994 - ERROR - Got signal 2
2017-10-17 01:35:03,559 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-17 01:35:03,575 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-17 01:36:00,901 - ERROR - Got signal 2 in MainProcess/MainThread
2017-10-17 01:36:00,901 - ERROR - Got signal 2
2017-10-17 01:36:01,032 - ERROR - Fast PoW got StopIteration
2017-10-17 01:37:24,122 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-17 01:37:24,122 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-17 18:15:54,022 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-17 18:15:54,023 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-17 20:17:22,828 - ERROR - Got signal 2 in MainProcess/MainThread
2017-10-17 20:17:22,829 - ERROR - Got signal 2
2017-10-17 20:17:28,388 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-17 20:17:28,388 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-17 21:32:07,644 - ERROR - Got signal 2 in MainProcess/MainThread
2017-10-17 21:32:07,644 - ERROR - Got signal 2
2017-10-17 21:32:10,163 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-17 21:32:10,164 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-17 22:07:25,447 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-17 22:07:25,454 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-17 22:07:25,938 - ERROR - There was a problem testing for a Namecoin daemon. Hiding the Fetch Namecoin ID button
2017-10-17 22:07:25,994 - WARNING - No notification.message plugin found
2017-10-17 22:15:52,234 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-17 22:15:52,236 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-17 22:15:52,615 - ERROR - There was a problem testing for a Namecoin daemon. Hiding the Fetch Namecoin ID button
2017-10-17 22:15:52,654 - WARNING - No notification.message plugin found
2017-10-17 22:16:06,640 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-17 22:16:06,641 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-17 22:16:12,980 - ERROR - Got signal 2 in MainProcess/MainThread
2017-10-17 22:16:12,982 - ERROR - Got signal 2
2017-10-17 22:17:42,349 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-17 22:17:42,351 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-17 22:30:25,743 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-17 22:30:25,744 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-17 22:32:35,638 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-17 22:32:35,640 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-17 22:50:30,265 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-17 22:50:30,266 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-17 23:08:31,805 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-17 23:08:31,806 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-17 23:08:55,684 - ERROR - Got signal 2 in MainProcess/MainThread
2017-10-17 23:08:55,684 - ERROR - Got signal 2
2017-10-17 23:22:03,233 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-17 23:22:03,233 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-17 23:33:37,756 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-17 23:33:37,756 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-17 23:39:41,940 - ERROR - Fast PoW got StopIteration
2017-10-17 23:41:06,568 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-17 23:41:06,569 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-17 23:56:06,496 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-17 23:56:06,497 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-17 23:57:28,275 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-17 23:57:28,276 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-17 23:58:16,607 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-17 23:58:16,608 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-17 23:59:38,517 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 252, in init
    bso = ctypes.WinDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
2017-10-17 23:59:38,519 - ERROR - C PoW test fail.
Traceback (most recent call last):
  File "C:\Users\VICTOR-PC\Documents\GitHub\taskhive-project\taskhive\src\bitmessage\proofofwork.py", line 262, in init
    bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
  File "C:\Python27\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found
